dp-game-webapp/app/about/page.tsx

```diff
@@ -0,0 +1,457 @@
+        </p>
+
+        <h1 className="my-2 text-lg sm:text-4xl font-bold tracking-tight">
+          Motivating Example
+        </h1>
+
+        <p className="mt-2">
+          Suppose we had access to a <b>database</b> that stores information
+          potentially sensitive data about people, such as if they have a
+          certain health condition. The database is operated by some trusted
+          source such as a health care provider, and it only allows{" "}
+          <b>aggregate</b> queries, say for the purpose of tracking prevalence
+          among the general population.
+        </p>
+        <p className="mt-2">
+          Now, let's say that <b>aggregation requirement</b> enforced by the
+          database results in an error if includes left than 100 people in the
+          query. For a single query, this is a pretty reasonable solution: for
+          example, a query of 100 people could tell us that {countTrue} people
+          have the condition. For everyone in that query, we'd uniformly learn
+          they all have a {countTrue}% chance of having the condition.
+        </p>
+        <DataTable
+          people={people.slice(0, -1)}
+          total={100}
+          countTrue={countTrue}
+        >
+          <h2 className="text-base font-semibold leading-6 text-gray-900 dark:text-gray-100">
+            Example Query Dataset 1
+          </h2>
+          <p className="mt-2 text-sm text-gray-700 dark:text-gray-300">
+            An example of a query of 100 rows of data with people and their
+            health condition status.
+          </p>
+        </DataTable>
+
+        <p className="mt-2">
+          Now, suppose we wanted to figure out if another person, Tali, has this
+          condition. Let's rerun the above query, but with one more person,
+          Tali. For demonstration, the following toggle controls value in the
+          database. Notice how the count changes exactly with her value.
+        </p>
+
+        <DataTable people={people} total={101} countTrue={countTrueWithTali}>
+          <h2 className="text-base font-semibold leading-6 text-gray-900 dark:text-gray-100">
+            Example Query Dataset 2
+          </h2>
+          <p className="my-2 text-sm text-gray-700 dark:text-gray-300">
+            An example of a query of 101 rows of data with people and their
+            health condition status.
+          </p>
+          <div className="mt-4 sm:ml-16 sm:mt-0 flex">
+            <h1 className="text-md mr-4 leading-6 text-gray-900 dark:text-gray-100">
+              Tali's status
+            </h1>
+
+            <Toggle
+              enabled={taliHasCondition}
+              setEnabled={setTaliHasCondition}
+            />
+          </div>
+        </DataTable>
+
+        <p className="mt-2">
+          This tells us, <b>definitively</b> Tali's status with respect to the
+          health condition. With just two queries, albeit carefully constructed,
```

Comment:
```suggestion
          health condition. With just two carefully constructed queries,
```

Reply:
too much? I feel like using "albeit" gives it a more conversational feel, where as "carefully constructed queries" feels more precise and academic.